server/app/routes/v1/certificates_api.rb

```diff
@@ -73,6 +73,20 @@ def register(data)
           end
         end
 
+        r.delete do
+          r.on ':subject' do |subject|
+            @certificate = @grid.certificates.find_by(subject: subject)
+            if @certificate
+              @certificate.destroy
```

Comment:
This needs to be a mutation that checks that the certificate is not in use by any services.

Reply:
true.

Reply:
fixed